Fix to Chrome inheritance bug (#22872)

prateekgoel opened the pull request:
I have changed:

```css
*,
*::before,
*::after {
  box-sizing: inherit; }
```
to

```css
*,
*::before,
*::after {
  box-sizing: border-box; }
```

Also, to address duplication of `box-sizing` from the `html` selector above, changed `box-sizing:border-box;` to `box-sizing:inherit;` from `html`.

Fixes #22872.
Resolves the issue with PR https://github.com/twbs/bootstrap/pull/23025.

prateekgoel committed:
Fix to Chrome inheritance bug (#22872)

mdo commented on scss/_reboot.scss:
```diff
@@ -19,7 +19,7 @@
 
 html {
-  box-sizing: border-box; // 1
+  box-sizing: inherit; // 1
```
This should just be removed outright, and the comments above updated.

prateekgoel replied:
Ok I am on it. I don't think we need to update comments. As the comments are still correct.

prateekgoel replied:
This is done. Let me know if any other changes are required.

prateekgoel committed:
Remove box-sizing from html

box-sizing removed from html and order of wild card selected and html selector swapped to maintain correctness of comments.

prateekgoel commented:
@mdo This is done. Let me know if any other changes are required.

mdo reviewed (approved):
Looks good, thanks! We'll get this in Beta 2.

prateekgoel commented:
Ok thanks.
